Masquerade4997Library/src/main/java/Library4997/MasqSensors/MasqOdometry.java

```diff
@@ -0,0 +1,29 @@
+package Library4997.MasqSensors;
+
+import Library4997.MasqMotors.MasqMotor;
+import Library4997.MasqResources.MasqHelpers.MasqHardware;
+
+/**
+ * Created by Keval Kataria on 11/13/2019
+ */
+public class MasqOdometry implements MasqHardware {
+    MasqMotor xSystem, ySystem;
+
+    public MasqOdometry(MasqMotor xSystem, MasqMotor ySystem) {
```

Comment:
Why is this the same constructor as the MasqPositionTracker. You should just pass in a static reference to the MasqPositionTracker.